grin/app/GrinCLI.hs

```diff
@@ -90,6 +90,11 @@ pipelineOpts =
   <|> flg (LVA PrintProgram) "print-lva-code" "Prints the live variable analysis machine"
   <|> flg (LVA RunPure) "run-lva-pure" "Runs the live variable analysis machine via pure interpreter"
   <|> flg (LVA PrintResult) "print-lva-result" "Prints the live variable analysis result"
+  <|> flg (ET Compile) "compile-et" "Compiles effect tracking analysis machine"
```

Comment:
This is getting out of hand :) In a later PR we should unify these steps, such as `--ai-compile=et`, `--ai-optimise=lva` etc.

No action required in this PR.

Reply:
Yes, I agree.